### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *syncthing* failed.

### ISSUE THREAD

**Issue by LizzieNya** [syncthing/website#44]
### <span aria-hidden="true">❌</span> Deploy Preview for *syncthing* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 4bc079740d044acaf8ed58fdd8a4906b748d3d02 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/syncthing/deploys/65f232d47a18de00087a65e8 |

_Originally posted by @netlify[bot] in https://github.com/syncthing/website/issues/42#issuecomment-1996062877_
